web/controllers/project_controller.ex

```diff
@@ -50,7 +50,7 @@ defmodule CodeCorps.ProjectController do
 
     case Repo.insert(changeset) do
       {:ok, project} ->
-        project = Repo.preload(project, [:categories, :organization, :skills])
+        |> Repo.preload([:categories, :organization, :posts, :skills])
```

Comment:
this is dying on semaphore. I don't think the `case` clause heads pass a value to its body, so I think it needs to be something like...
```elixir
project =
  project
  |> Repo.preload([:categories, :organization, :posts, :skills])
  ...
```
and then again the same thing around line 74